assets/source/setup-guide/app/helpers/documentation-link-props.js

```diff
@@ -21,17 +33,18 @@ import { recordEvent } from '@woocommerce/tracks';
  *
  * Please be careful not to overwrite the `onClick` handler coincidently.
  *
- * @fires wcadmin_pfw_documentation_link_click on click, with given `linkId` and `context`.
+ * Notice documentation_link_click is the default eventName and pfw_ is not required
  *
+ * @fires wcadmin_pfw_documentation_link_click on click, with given `linkId` and `context`.
  * @param {Object} props React props.
  * @param {string} props.href Href to used by link and in track event.
- * @param {string} props.linkId Forwarded to {@link wcadmin_pfw_documentation_link_click}
- * @param {string} props.context Forwarded to {@link wcadmin_pfw_documentation_link_click}
+ * @param {string} props.linkId Forwarded to {@see wcadmin_pfw_documentation_link_click}
+ * @param {string} props.context Forwarded to {@see wcadmin_pfw_documentation_link_click}
  * @param {string} [props.target='_blank']
  * @param {string} [props.rel='noopener']
  * @param {Function} [props.onClick] onClick event handler to be decorated with firing Track event.
+ * @param {string} props.eventName The name of the event to be recorded
```

Comment:
It's optional sou we can mark it in the docs
```suggestion
 * @param {string} [props.eventName='documentation_link_click'] The name of the event to be recorded
```

Also, as mentioned in another comment, I'd use the full path, to follow the field guide about Tracks naming convention.

Reply:
Fixed here https://github.com/woocommerce/pinterest-for-woocommerce/pull/299/commits/4806fb3f592054a5e2683f7208b1f6e0efaabed3